packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -17,6 +17,17 @@
             "allowedValues": ["Create", "Update"],
             "description": "SSM Publish Document operation type"
         },
+        {
+            "name": "stepFunctionsSyncType",
```

Comment:
all of these stepfunctions metrics need to be upstreamed to https://github.com/aws/aws-toolkit-common

they have been in this override file for a long time

Reply:
It have already been raised in one of the previous pull requests, we are still developing the feature and we will most probably need to add more metrics, which is more handy to do in a single package.

This approach also aligns with what is written in [telemetry](https://github.com/aws/aws-toolkit-vscode/blob/master/docs/telemetry.md) readme:
> When your feature is released, the "development" metrics you defined in vscodeTelemetry.json should be upstreamed to [aws-toolkit-common](https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json).

We have a SIM in our project to transfer metrics to that package closer to launch. Can we keep them in the current package for now?

Reply:
> Can we keep them in the current package for now?

What is the ECD?

Reply:
we are planning to launch around end of February